src/diffpy/pdfgui/gui/pdfguiglobals.py

```diff
@@ -16,8 +16,9 @@
 """This module contains global parameters needed by PDFgui."""
 
 import os.path
+from importlib import resources
 
-from pkg_resources import Requirement, resource_filename
+from pkg_resources import Requirement
```

Comment:
I think pkg_resources is deprecated so we want to get rid of this.  I would put pkg_resources fixes on a separate PR tbh.